sku.0/sys.server/compiled/game/script/player/player_beastmaster.java

```diff
@@ -669,7 +669,13 @@ public void reviveBeast(obj_id self, obj_id beast) throws InterruptedException
         }
         float revivePercent = 10;
         float maxHealthFloat = getMaxAttrib(beast, HEALTH);
-        revivePercent += getEnhancedSkillStatisticModifierUncapped(self, "expertise_bm_pet_recovery");
+        if (hasSkill(self, "expertise_bm_pet_recovery_3")) {
+            revivePercent = 75;
+        } else if (hasSkill(self, "expertise_bm_pet_recovery_2")) {
+            revivePercent = 50;
+        } else if (hasSkill(self, "expertise_bm_pet_recovery_1")) {
+            revivePercent = 25;
+        }

```

Comment:
The existing skill mod check and percentage increase seems to work fine here, does it not? When I tested this in-game the modifier correctly adjusted without this change.

This isn't the way we'd fix this either way, because the problem would be with the skill modifier. Skill checks are redundant and use more resources when the skill modifier value is what we need to determine to revive percentage.